frontend/packages/console-shared/src/components/popover-body/popover-body.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Dropdown } from '@console/internal/components/utils/dropdown';
+import { Link } from 'react-router-dom';
+import { DataPoint } from '@console/internal/components/graphs';
+import './popoverBody.scss';
+import { ExternalLink, Humanize, resourcePathFromModel } from '@console/internal/components/utils';
+import { K8sKind } from '@console/internal/module/k8s';
+import { PodModel, NodeModel, ProjectModel } from '@console/internal/models';
+import { connectToURLs, MonitoringRoutes } from '@console/internal/reducers/monitoring';
+import { getPrometheusExpressionBrowserURL } from '@console/internal/components/graphs/prometheus-graph';
+
+const dropdownKeys = ['By Pods', 'By Node', 'By Projects'];
+const models = [PodModel, NodeModel, ProjectModel];
+const dropdownItems = _.zipObject(dropdownKeys, dropdownKeys);
+
+export const PopoverBody = connectToURLs(MonitoringRoutes.Prometheus)(
+  React.memo(
+    ({ topConsumers, title, humanize, query, urls }: React.PropsWithoutRef<PopoverBodyProps>) => {
+      const [selectedFilter, setSelectedFilter] = React.useState(dropdownKeys[0]);
+      const [bodyData, setBodyData] = React.useState(null);
+
+      const consumer = title
+        ? title.padStart(title.length + 1, ' ').padEnd(title.length + 2, ' ')
```

Comment:
Can we simplify this code?

Is the intention to simply pad `title` with one space before and after?